docs/learning/quantity/index.md

```diff
@@ -43,6 +45,14 @@ For a detailed description of all the setting supported by FormatProp see the EC
 
 ### Concepts
 
+#### Formats Provider
+
+A [FormatsProvider]($ecschema-metadata) interface helps provide all the necessary formats for displaying formatted quantity values, while also enabling users to add formats of their own.
+
+A [MutableFormatsProvider]($ecschema-metadata) interface extends the read-only FormatsProvider above by allowing adding or removing Formats to the provider.
```

Comment:
```suggestion
A [MutableFormatsProvider]($ecschema-metadata) interface extends the read-only [FormatsProvider]($ecschema-metadata) above by allowing adding or removing Formats to the provider.
```

Reply:
I've wrapped the term in backtick instead, because the term already uses a link in the relevant section